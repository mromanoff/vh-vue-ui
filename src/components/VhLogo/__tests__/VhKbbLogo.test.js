import { shallowMount } from '@vue/test-utils';
import VhKbbLogo from '../VhKbbLogo.vue';

describe('VhKbbLogo.vue', () => {
  let wrapper;

  it('renders default VhKbbLogo', () => {
    wrapper = shallowMount(VhKbbLogo);

    expect(wrapper).toMatchSnapshot();
  });
});
