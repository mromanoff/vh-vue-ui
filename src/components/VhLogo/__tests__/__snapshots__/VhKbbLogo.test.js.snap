// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VhKbbLogo.vue renders default VhKbbLogo 1`] = `<div><img src="https://www.vehiclehistory.com/images/logo_kbb_vert_110x201.png" alt="KBB Logo" class="KbbLogo"></div>`;
